src/run_tribler_headless.py

```diff
@@ -146,7 +175,7 @@ def main(argv):
     coro = service.start_tribler(args)
     ensure_future(coro)
 
-    if sys.platform == 'win32':
+    if sys.platform == 'win32' and sys.version < 3.8:
```

Comment:
> sys.version < 3.8

This results in `TypeError: '<' not supported between instances of 'str' and 'float'`. I'd suggest to use to add a check for major and minor versions using `sys.version_info` like:
```
sys.version_info < (3, 8)
```